README.md

```diff
@@ -1 +1,196 @@
+- [⭐️ Show your support](#support)
+- [🔭 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+This application allows you to:
+
+- See a random greeting
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technology</summary>
+  <ul>
+    <li>Ruby</li>
+    <li>Rails</li>
+    <li>PostgresSQL</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li>VS Code</li>
+    <li>GIT</li>
+    <li>GITHUB</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Use Postgres database.
+- Use React to fetch the API
+- Use WebPack to pack the JavaScript code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Sorry, Currently not available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+you have to those tools in your local machine.
+
+- [ ] Ruby
+- [ ] Rails
+- [ ] PostgresSQL
+- [ ] GIT & GITHUB
+- [ ] Any Code Editor (VS Code, Brackets, etc)
+
+### Clone Repo
+
+Clone the project using the following bash command in an appropriate location.
+
+```bash
+  git clone https://github.com/PierreBarba/Ruby-on-Rails-React.git
+```
+
+Go to the project directory.
+
+```bash
+  cd Ruby-on-Rails-React
+```
+
+To learn more [check Custom Credentials](https://edgeguides.rubyonrails.org/security.html#custom-credentials) documentation.
+
+### Add necessary packages
+
+For installing necessary packages, run the following bash command:
+
+```bash
+  bundle install
+```
+
+### Run the server in development mode
+
+In the project directory, you can run the project by using following bash command:
+
+```bash
+  ./bin/dev
+```
+
+And now you can visit the site with the URL http://localhost:3000
+
+### Run the test
+
+For testing you can run:
+
+```bash
+  bundle exec rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+**Jean Pierre Barba**
+
+[![portfolio](https://img.shields.io/badge/my_portfolio-000?style=for-the-badge&logo=ko-fi&logoColor=white)](https://github.com/PierreBarba) [![linkedin](https://img.shields.io/badge/PierreBarba-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/jean-pierre-barba/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Live demo will added soon**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../../issues/).
+
```

Comment:
- [ ] As requested by the previous reviewer please make sure to fix the issues page link. It's still broken
 - The correct URL should be `https://github.com/PierreBarba/Ruby-on-Rails-React/issues`